jenkins/jobs/integration_tests.pipeline

```diff
@@ -52,7 +52,7 @@ pipeline {
     OS_AUTH_URL="https://kna1.citycloud.com:5000"
     OS_USER_DOMAIN_NAME="CCP_Domain_37137"
     OS_PROJECT_DOMAIN_NAME="CCP_Domain_37137"
-    OS_REGION_NAME="Kna1"
+    OS_REGION_NAME="${OS_REGION_NAME}"
```

Comment:
1. This is setting `OS_REGION_NAME` to `OS_REGION_NAME`, ie. it does nothing.
2. Other pipeline files set it to `OS_REGION`?